docset/winserver2012-ps/bitlocker/Remove-BitLockerKeyProtector.md

```diff
@@ -24,38 +24,54 @@ Remove-BitLockerKeyProtector [-MountPoint] <String[]> [-KeyProtectorId] <String>
 The **Remove-BitLockerKeyProtector** cmdlet removes a key protector for a volume protected by BitLocker Drive Encryption.
 
 You can specify a key protector to remove by using an ID.
-To add a protector, use the Add-BitLockerKeyProtector cmdlet.
+To add a protector, use the **Add-BitLockerKeyProtector** cmdlet.
 
 If you remove all the key protectors for a BitLocker volume, BitLocker stores the data encryption key for the volume without using encryption.
 This means that any user that can access the volume can read the encrypted data on the volume unless you add a key protector.
 Any encrypted data on the drive remains encrypted.
 
 We recommend you have at least one recovery password as key protector to a volume in case you need to recover a system.
 
-For an overview of BitLocker, see BitLocker Drive Encryption Overviewhttp://technet.microsoft.com/en-us/library/cc732774.aspx (http://technet.microsoft.com/en-us/library/cc732774.aspx) on TechNet.
+For an overview of BitLocker, see [BitLocker Drive Encryption Overview](https://technet.microsoft.com/en-us/library/cc732774.aspx).
```

Comment:
Compare the content, but you'll probably want to update this to a non-TN link, i.e. https://docs.microsoft.com/windows/security/information-protection/bitlocker/bitlocker-overview.

Reply:
The updated link is for Windows 10 and probably is not appropriate for Server 2012 / 2012 R2 articles. I could not find a non-TN link for it, unfortunately.

Reply:
OK, that's fine. Can you use this link in its place?

https://docs.microsoft.com/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc732774(v=ws.11)

Reply:
@tseward I guess I have not looked properly in that section. Now I think I found an even better one:
https://docs.microsoft.com/previous-versions/windows/it-pro/windows-8.1-and-8/dn641993(v=ws.11)